python/paddle/fluid/parallel_executor.py

```diff
@@ -22,10 +22,14 @@
 
 class ParallelExecutor(object):
     def __init__(self,
-                 loss_name,
-                 use_cuda,
+                 loss_name=None,
+                 use_cuda=None,
                  num_threads=None,
-                 allow_op_delay=False):
+                 allow_op_delay=False,
+                 main_program=None,
+                 startup_program=None,
+                 local_scopes=None,
+                 run_startup=True):
```

Comment:
If startup_program is None, then startup is not run?

Reply:
`startup_program` is always used, even for the parallel testing, the code is here https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/framework/parallel_executor.cc#L69